### DISCUSSION TITLE
Is it possible to create a placeholder chart?

### DISCUSSION THREAD

**Discussion by Benyuss** [patrykandpatrick/vico#575] [Questions] [answered]
I'd like to have a straight line in the chart when there are no values.
I tried to pujt a fake list into the component with values of (0f, 0f) and even (1f,1f) but when I do that the chart won't be rendered at all.
I figure it's because LineChart.505 (getDrawY) only makes a valid point if it can create a valid value. Is it possible to override this behavior? Am I missing something?
I'm on 1.14 beta, should I switch over to 2.0 alpha to have some of the latest features which might support this behaviour?
Thanks!

**Comment by patrickmichalik** [comment]
Hello! Just to confirm, when you use the notation (0, 0), are you referring to a single entry with both coordinates equal to zero or two entries whose _y_ values are both zero? A line is drawn when there are two or more points, so the latter should give you a horizontal line. If it doesn’t, could you share the relevant portion of your code?

Here’s a quick example. Depending on the specifics of the desired result, some changes might be needed, but a horizontal line is drawn.

```kt
val chartEntryModel = entryModelOf(0, 0)
```

```kt
Chart(
    chart = lineChart(),
    model = chartEntryModel,
    startAxis = rememberStartAxis(),
    bottomAxis = rememberBottomAxis(),
)
```

<img src="https://github.com/patrykandpatrick/vico/assets/120058021/fd1966a9-7e37-4672-9120-6ee2dadabd10" width="720" />

**Reply by Benyuss** [reply]
Hello @patrickmichalik,
Thank you for the quick answer, really appreciate it!

Seems like there's some problem in my implementation, I'll take care of it then.
Is there a way to create a chart wwhere all the datapoints are centered?
Like they are not on the top or the very bottom but these two placeholders are in the middle. Also I've seen many comments about getting rid of the start and end paddings, do you have a solution in mind for that?
Thanks!

**Reply by patrickmichalik** [reply, marked as answer]
You’re welcome, @Benyuss! To center the line, you can use `AxisValuesOverrider`. As for the padding, `HorizontalLayout.FullWidth` removes it. Because the maximum zoom factor is 10 (this will be made customizable in Vico 2.0.0 Alpha 9), the line might not reach the very end of the chart. To combat this, you can decrease the _x_ step.

```kt
val chartEntryModel = entryModelOf(0, 0)
val axisValueOverrider = AxisValuesOverrider.fixed(minY = -1f)
val horizontalLayout = HorizontalLayout.fullWidth()
```

```kt
Chart(
    chart = lineChart(axisValuesOverrider = axisValueOverrider),
    model = chartEntryModel,
    startAxis = rememberStartAxis(),
    bottomAxis = rememberBottomAxis(),
    horizontalLayout = horizontalLayout,
    getXStep = { 0.2f },
)
```

<img src="https://github.com/patrykandpatrick/vico/assets/120058021/e01f6e01-2074-4c55-9f78-ba00e2aaa1a4" width="720" />
